admin-tools-command.php

```diff
@@ -18,3 +18,19 @@
 }
 
 EE::add_command( 'admin-tools', 'Admin_Tools_Command' );
+
+
+EE::add_hook('before_invoke:admin-tools up', 'init_global_auth' );
```

Comment:
Same issue:
https://github.com/EasyEngine/mailhog-command/pull/4#discussion_r214965570

Execute auth init in up function only: https://github.com/EasyEngine/admin-tools-command/blob/f3d649627974fcf9e6b2ded54b28eea83c1359aa/src/Admin_Tools_Command.php#L102